docker-compose.yml

```diff
@@ -1,53 +1,83 @@
 version: '2'
 services:
 
-  postgres:
-    image: postgres
+  database:
+    image: camptocamp/postgis:9.5
```

Comment:
I'd rather stick to official image, since postgis is not required here to my knowledge.